Lib/test/test_collections.py

```diff
@@ -1867,6 +1867,63 @@ def test_init(self):
         self.assertRaises(TypeError, Counter, (), ())
         self.assertRaises(TypeError, Counter.__init__)
 
+    def test_order_preservation(self):
+        # Input order dictates items() order
+        self.assertEqual(list(Counter('abracadabra').items()),
+               [('a', 5), ('b', 2), ('r', 2), ('c', 1), ('d', 1)])
+        # letters with same count:   ^----------^         ^---------^
+
+        # Verify retention of order even when all counts are equal
+        self.assertEqual(list(Counter('xyzpdqqdpzyx').items()),
```

Comment:
IMO this test is same than above. what about use one of the?

Reply:
It's a similar test, but in this case all the inputs are equal.  In the previous test, it was mixed.